doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -1,18 +1,605 @@
+    
+    | a   | b   | value |
+    | --- | --- | ----- |
+    | a0  | b0  | 0     |
+    | a0  | b1  | 1     |
+    | a1  | b0  | 2     |
+    | a1  | b1  | 3     |
+    
+    The name of the column containing the data can changed by setting the argument ``value_name``.
+
+
+- code: |
+    arr = ndtest((2, 2))
+    
+    # dump the array arr in a narrow format (one column per axis plus one value column)
+    # by default, the name of the column containing data is set to 'value'
+    arr.to_excel('arrays.xlsx', 'array_narrow', wide=False)
+    
+    # dump the array arr in a narrow format and set the name of the column 
+    # containing data to 'data'
+    arr.to_excel('arrays.xlsx', 'array_narrow', wide=False, value_name='data')
+
+  id: 4
+
+- markdown: |
+    ##### Exporting only data
+    
+    By setting the argument ``header`` to False, only the data are exported while axes names and labels are dropped.
+
+
+- code: |
+    arr = ndtest((2, 2))
+    
+    # drop axes names and labels. Only data are exported
+    arr.to_excel('arrays.xlsx', 'data_only', header=False)
+
+  id: 5
+
+- markdown: |
+    #### Dumping Arrays to Excel Sheets with open_excel 
+
+
+- markdown: |
+    ##### Basic Usage
+    
+    ``open_excel`` must be used with the Python keyword ``with``. If the Excel file doesn't exist, the argument ``overwrite_file`` must set to True. The method ``save`` must be called to actually write data in the Excel file. 
+    For example: 
+
+
+- code: |
+    arr = ndtest((2, 2))
+    arr2 = zeros((2, 2))
+    arr3 = ones((2, 2))
+    
+    # to create a new Excel file, argument overwrite_file must be set to True
+    with open_excel('new_excel_file.xlsx', overwrite_file=True) as wb:
+        # add a new sheet 'arr' and dump the array 'arr' in it 
+        wb['arr'] = arr.dump()
+        # add a new sheet 'arr2' and dump the array 'arr2' in it 
+        wb['arr2'] = arr2.dump()
+        # add a new sheet 'arr3' and dump the array 'arr2' in it 
+        wb['arr3'] = arr3.dump()
+        # actually write data in the Workbook
+        wb.save()
+        
+    # the Workbook is automatically closed when getting out the block 
+    # defined by the with statement
+    print('dumping arrays arr, arr2 and arr3 is done')
+
+  id: 6
+
+- markdown: |
+    ##### Specifying Position
+    
+    It is possible to choose the position of the first cell containing the data of the dumped array. This allows to export several arrays in the same sheet: 
+
+
+- code: |
+    arr = ndtest((2, 2))
+    arr2 = zeros((2, 2))
+    arr3 = ones((2, 2))
+    
+    with open_excel('new_excel_file.xlsx') as wb:
+        # add a new sheet 'arrays' and dump the array 'arr' starting at cell 'A1' 
+        wb['arrays'] = arr.dump()
+        # dump the array 'arr2' in sheet 'arrays' starting at cell 'A5' 
+        wb['arrays']['A5'] = arr2.dump()
+        # dump the array 'arr3' in sheet 'arrays' starting at cell 'A9' 
+        wb['arrays']['A9'] = arr3.dump()
```

Comment:
also show that if you have many arrays to store on the same sheet, you can store the sheet in a variable and use that?

```python
sheet = wb['arrays']
sheet['A5'] = arr2.dump()
```